css/style.css

```diff
@@ -549,27 +557,51 @@ button.button-inline:hover {
 #card-dates {
 	font-size: 80%;
 	color: #aaaaaa;
-	margin-bottom: 3px;
-}
-
-#card-meta #labels {
-	margin-bottom: 10px;
+	text-align: right;
 }
 
 #card-meta .duedate {
 	display: flex;
+	align-content: center;
+	min-height: 38px;
+}
+
+#card-meta .duedate .timepicker-input {
+	width: 50px;
+	margin: 3px;
+}
+
+#card-meta .duedate .datepicker-input {
+	width: 85px;
+}
+
+#card-meta .duedate .icon {
+	opacity: 0.3;
+}
+
+#card-meta .duedate .icon:hover {
+	opacity: 0.7;
```

Comment:
but it doesn't change opacity. only color. for buttons we use always a different opacity on mouse over.